getting-started/web-app.md

```diff
@@ -0,0 +1,325 @@
+
+1. Install the Telerik Blazor NuGet package:
+
+  1. Select the `telerik.com` **Package source** that you [added earlier](#step-2-add-the-telerik-nuget-feed-to-visual-studio). As this is a private NuGet feed, you must authenticate with your [Telerik account](https://www.telerik.com/account/) user name and password.
+  1. Select the **Browse** tab, find the NuGet package, and click **Install**. Commercial license holders should install `Telerik.UI.for.Blazor`. Trial users should install `Telerik.UI.for.Blazor.Trial`.
+
+## Step 4: Enable the Blazor UI Components
+
+To enable the Telerik UI for Blazor components, you must add several client-side dependencies to the application:
+
+### 4.1. Add the Telerik UI for Blazor Client Assets
+
+1\. Add the `telerik-blazor.js` file to your `App.razor` file as a [static asset]({%slug getting-started/what-you-need%}#using-static-assets) or from a [CDN]({%slug getting-started/what-you-need%}#using-cdn).
+
+**HTML**
+
+@[template](/_contentTemplates/common/js-interop-file.md#js-interop-file-snippet)
+
+2\. To select the appearance and color scheme for the Telerik Blazor components, add the [theme stylesheet]({%slug general-information/themes%}) in your `App.razor` file. Reference it as a [static asset]({%slug general-information/themes%}#static-assets) or from a [CDN]({%slug general-information/themes%}#cdn).
+
+3\. (optional - applicable if you are using static assets) To enable the use of static assets in your project, add the `app.UseStaticFiles();` line to the `Program.cs` of your Blazor Server project (by default, this line is already present).
+
+**C#**
+@[template](/_contentTemplates/common/js-interop-file.md#enable-static-assets-snippet)
+
+###  4.2. Include @using Statements
+
+In the `~/_Imports.razor` file, add the `@using` directives below. This configures the project to recognize the Telerik components in all files. You can register one or both icon namespaces, depending on the [icon type you will be using]({%slug general-information/font-icons%}).
+
+**_Imports.razor**
+    
+    @using Telerik.Blazor
+    @using Telerik.Blazor.Components
+    @using Telerik.FontIcons
+    @using Telerik.SvgIcons
+
+###  4.3. Add the TelerikRootComponent
+
+Add a `TelerikRootComponent` component as a top-level component in the app and make sure it wraps all content.  Add the component in the preferred layout file - could be the `MainLayout.razor` or a custom layout file. The important part is to ensure that the layout component where the `TelerikRootComponent` is defined has interactive mode.
+
+.NET 8.0 introduces [new render modes for the Blazor components](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0). The `TelerikRootComponent` must be placed in a layout page (e.g. `MainLayout.razor`) with enabled interactive mode. At the time of writing, the default render mode is `Static` and not interactive, so you need to make this change explicitly in your app.
```

Comment:
This paragraph repeats the last sentence from the previous one. Consider refactoring or merging.